sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/pom.xml

```diff
@@ -50,6 +50,12 @@
     </dependency>
 
     <!-- Test dependencies -->
+    <dependency>
```

Comment:
(can't comment on line no.23 and 24) - we should update the name and description of the artifact to not limit this package  to "Azure Redis". This should work with any Redis service.